src/main/java/org/springframework/data/neo4j/config/AbstractNeo4jConfig.java

```diff
@@ -47,6 +48,9 @@ public abstract class AbstractNeo4jConfig extends Neo4jConfigurationSupport {
 	@Autowired
 	private ObjectProvider<UserSelectionProvider> userSelectionProviders;
 
+	@Autowired
```

Comment:
Used field injection here (and reactive client) to avoid forced change of the `Neo4jClient` bean parameters.

Reply:
🙈